config/core/configmaps/observability.yaml

```diff
@@ -45,9 +45,7 @@ data:
 
     # logging.revision-url-template provides a template to use for producing the
     # logging URL that is injected into the status of each Revision.
-    # This value is what you might use the the Knative monitoring bundle, and provides
-    # access to Kibana after setting up kubectl proxy.
-    logging.revision-url-template: "http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))"
+    logging.revision-url-template: "http://logging.example.com/?revisionUID=${REVISION_UID}"
```

Comment:
I wonder whether we should use empty here. Usually an example value in configmap is the default value that works. Apparently this one doesn't work.

Reply:
I added some logic to allow the default and the example to differ

I think it's fine for the example to highlight the revision uid substitution while the default is empty